index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
```

Comment:
Started Reviewing the code

Reply:
The problem here is that too many Tetris pieces are the same color. In your review, you might want to suggest that I choose colors which would be easier to see when playing the game.